extensions/wikia/ARecoveryEngine/templates/ARecoveryEngineApiController_getBootstrap.mustache

```diff
@@ -4,11 +4,16 @@ window._sp_ = {
     config: {
         enable_rid: true,
         content_control_callback: function() {
-            var msg = '<h2>Ad blocker interference detected!</h2>';
+            var logGroup = 'ext.wikia.aRecoveryEngine.recovery.bootstrap',
+                msg = '<h2>Ad blocker interference detected!</h2>';
```

Comment:
What about making the same message [here](https://github.com/Wikia/app/blob/155c0f83b95b1e26c7e21796ce1fe6af347087ba/extensions/wikia/ARecoveryEngine/css/recoveryLock.scss#L7)?